core/src/main/java/org/eclipse/hono/util/ResourceIdentifier.java

```diff
@@ -230,13 +230,25 @@ public String toString() {
     /**
      * Gets a string representation of this resource identifier's
      * <em>endpoint</em> and <em>tenantId</em>.
-     * 
+     *
      * @return A string consisting of the properties separated by a forward slash.
      */
     public String getBasePath() {
         return basePath;
     }
 
+    /**
+     * Gets a string representation of ths resource identifiers's parts without the base path {@see getBasePath()}.
```

Comment:
typo, should read `identifiers'`. Also, we usually put an empty line between the description and the tags. `@see` should also go down to the tags section FMPOV.